.github/workflows/nightly.yml

```diff
@@ -140,7 +140,9 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ needs.create-nightly-tag.outputs.tag }}
-          persist-credentials: false
+          # To limit the ways a secret can be leaked, it should not be kept
+          # in the repository, but in this case we need git commands to work.
```

Comment:
The wording on this comment (and all other instances of it) is unclear/confusing to me. I would phrase it as something like
`Save the access token to the local git config, so later git commands can work.`

Reply:
Thanks. Updated!